src/database/config/config.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config();
+
+module.exports = {
+  development: {
+    url: 'postgres://postgres:babu02@127.0.0.1:5432/ecdev_db',
```

Comment:
remove your database url and use environment variable to allow everyone
url: 'proccess.env.DEV_DATABASE_URL'

Reply:
ooh!!! I forget to change, thanks @leandreAlly